argoproj-crd/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: argoproj-crd
-version: 0.1.2
-appVersion: 1.6.2
+version: 0.1.3
```

Comment:
@taishin 
`0.2.0`？
It is better to change the minor version of chart to add features.